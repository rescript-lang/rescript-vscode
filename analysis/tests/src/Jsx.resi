@react.component
let make: (~first: string) => React.element
//  ^hov

let y: int
//  ^hov

// type t = React.e
//                 ^com

// let x : React.e
//                ^com
