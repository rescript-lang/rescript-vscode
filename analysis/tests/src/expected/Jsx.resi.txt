Hover src/Jsx.resi 1:4
{"contents": "```rescript\n(~first: string, ~key: string=?, unit) => {\"first\": string}\n```"}

Hover src/Jsx.resi 4:4
{"contents": "```rescript\nint\n```"}

Complete src/Jsx.resi 7:19
posCursor:[7:19] posNoWhite:[7:18] Found type:[7:12->7:19]
Ptyp_constr React.e:[7:12->7:19]
Completable: Cpath Type[React, e]
[{
    "label": "element",
    "kind": 22,
    "tags": [],
    "detail": "type element",
    "documentation": null
  }]

Complete src/Jsx.resi 10:18
posCursor:[10:18] posNoWhite:[10:17] Found type:[10:11->10:18]
Ptyp_constr React.e:[10:11->10:18]
Completable: Cpath Type[React, e]
[{
    "label": "element",
    "kind": 22,
    "tags": [],
    "detail": "type element",
    "documentation": null
  }]

